docs/controller-skills/military.md

```diff
@@ -30,24 +30,33 @@ Outside controlled airspace, it is the responsibility of the controller to provi
 
 For aircraft unfamiliar with the restricted area, provide *suggested* headings to avoid. Get the aircraft to squawk ident to temporarily identify them. Remember that you cannot vector aircraft OCTA.
 
-### CTA
-You must separate all aircraft in controlled airspace by *half the applicable lateral standard* from a Restricted Area (ie, 1.5nm for TCU, 2.5nm for Enroute), and 1000ft vertically.
+#### CTA
+You must separate all aircraft in controlled airspace by *half the applicable lateral standard* from a Restricted Area (ie, **1.5nm** for TCU, **2.5nm** for Enroute). Aircraft operating at the vertical limits of Restricted Areas are considered to be separated from it.
 
-## Activation times
+### Activation times
 Some Restricted Areas are not always active. Some may be active only on certain days, or certain hours of the day. More info can be found in the [ERSA](https://www.airservicesaustralia.com/aip/current/ersa/PRD__01DEC2022.pdf){target=new}, or on the Restricted Areas window in vatSys
 
-### NOTAM
+#### NOTAM
 Some Restricted Areas will be activated by NOTAM, most commonly for use by RAAF Virtual. These are available on the [VATPAC NOTAMs Page](https://vatpac.org/publications/notam){target=new}. These Restricted Areas will automatically become active on vatSys with AIRAC updates, so they are visually displayed on the screen.
 
-##### Example
+###### Example
 <figure markdown>
 ![Example Restricted Area Activation NOTAM](img/vsoanotam.png){ width="800" }
   <figcaption>Example Restricted Area Activation NOTAM</figcaption>
 </figure>
 
-### Aircraft operating inside Restricted Area
+#### Aircraft operating inside Restricted Area
 Military aircraft are permitted to operate within their designated Restricted Area as per the relevant NOTAM. Even if this Restricted Area is inside your airspace, you have no control authority over it.
 
 !!! example
     **ARA**: "DRGN379, shortly entering R604 Restricted Area, identification and control services terminated, frequency change approved. Contact me prior to leaving for airways clearance"  
-    **DRGN379**: "Wilco, DRGN379"
\ No newline at end of file
+    **DRGN379**: "Wilco, DRGN379"
+
+## Formation Flights
+Reserved.
```

Comment:
No action required now, but when we get around to filling this in, might be worth putting it in Controller Skills somewhere and linking this section to that, so we can cover civil formation ops too. Pilot SOPs will cover formation flying as well.

Reply:
That quoting didn't work very well, was talking about the Formation section